influxql/result.go

```diff
@@ -40,6 +41,18 @@ type Message struct {
 	Text  string `json:"text"`
 }
 
+// ReadOnlyWarning generates a warning message that tells the user the command
+// they are using is being used for writing in a read only context.
+//
+// This is a temporary method while to be used while transitioning to read only
+// operations for issue #6290.
+func ReadOnlyWarning(stmt string) *Message {
+	return &Message{
+		Level: WarningLevel,
+		Text:  fmt.Sprintf("deprecated use of '%s' in a read only context, please use a POST request instead", stmt),
```

Comment:
Where does this message get returned?  Is it logged by the server or returned in the response body somewhere?